ironfish/src/memPool/memPool.ts

```diff
@@ -118,18 +118,19 @@ export class MemPool {
     beforeSize: number,
     seenNullifiers: Nullifier[],
   ): Promise<boolean> {
-    // it's faster to check if spends have been seen or not, so do that first
     for (const spend of transaction.spends()) {
       const verificationError = await this.chain.verifier.verifySpend(spend, beforeSize)
       if (verificationError) {
         return false
       }
     }
+
     const validity = await transaction.verify()
     if (!validity.valid) {
       return false
     }
 
+    // it's faster to check if spends have been seen or not, so do that first
```

Comment:
looks like we're not doing that first anymore

Reply:
we never were, deleted comment.